toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -315,7 +408,7 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 	}
 
 	hostname := config.Hostname
-	if !isRootFS {
+	if !isRootFS && mountPointToFsTypeMap[rootMountPoint] != "overlay" {
```

Comment:
Since the literal string "overlay" is used in a few places, I'd recommend replacing it with a const instead.

Reply:
Fixed.

---
In reply to: [573076804](https://github.com/microsoft/CBL-Mariner/pull/611#discussion_r573076804) [](ancestors = 573076804)